sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -29,15 +29,22 @@
 import logging
 import os
 import tempfile
+import warnings
 from collections.abc import Iterable
+from pathlib import PurePath
 
 import apache_beam as beam
+from apache_beam.pipeline import Pipeline
+from apache_beam.runners import DataflowRunner
 from apache_beam.runners import runner
+from apache_beam.runners.direct import direct_runner
 from apache_beam.runners.interactive import cache_manager as cache
+from apache_beam.runners.interactive.interactive_runner import InteractiveRunner
```

Comment:
I would recommend moving this import to where it's used instead of putting it at the top of the module to avoid a dependency loop between this module and the interactive_runner module.

Reply:
I've moved this line into the region of `get_cache_manager` where we use the InteractiveRunner now, thanks!